spec/integration/driver_spec.rb

```diff
@@ -860,6 +860,23 @@ def create_screenshot(file, *args)
       end
     end
 
+    it 'allows the driver to have a custom host' do
```

Comment:
use the RSpec provided `skip` to skip the test so a notice gets printed that a test was skipped